packages/@react-aria/menu/src/useMenuTrigger.ts

```diff
@@ -37,9 +37,9 @@ export function useMenuTrigger(props: MenuTriggerProps, state: MenuTriggerState)
   });
 
   let onPress = () => {
-    if (!isDisabled && !state.isOpen) {
+    if (!isDisabled) {
       state.setFocusStrategy('first');
-      state.setOpen(true);
+      state.setOpen(!state.isOpen);
```

Comment:
This is so you can double tap on the trigger that opened the menu to close it, since voiceover does not move focus there.

Reply:
can we make this a comment in the code?

Reply:
not sure it makes sense as a code comment. it's just toggling on press. this is more of a description of why it changed.